packages/thirdweb/src/chainsaw/urls.ts

```diff
@@ -0,0 +1,47 @@
+import { getThirdwebDomains } from "../utils/domains.js";
+
+export const getChainsawBaseUrl = (): string => {
+  const domain: string = getThirdwebDomains().chainsaw;
+  return `${domain}/v2`;
```

Comment:
This might get confusing to wrap the `getThirdwebDomains` result

Reply:
We want to "force" the v2 path since all other endpoints will be deprecated in the future.

But renamed the function to have more clarity [4c66b23](https://github.com/thirdweb-dev/js/pull/3741/commits/4c66b237be7a81011db6cc502fa533d3484169f7)